File: app/src/main/java/com/BlizzardArmory/diablo/Item/OneOf.java
package com.BlizzardArmory.diablo.Item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class OneOf {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("textHtml")
    @Expose
    private String textHtml;

    public String getText() {
        return text;
    }

    public String getTextHtml() {
        return textHtml;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/item/OneOf.kt
package com.BlizzardArmory.model.diablo.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type One of.
 */
@Keep
data class OneOf(

    @SerializedName("text")

    val text: String,


    @SerializedName("textHtml")

    val textHtml: String

)